dev-portal/public/apigateway-js-sdk/apigClient.js

```diff
@@ -53,9 +53,9 @@ apigClientFactory.newClient = function (config) {
     }
 
     // extract endpoint and path from url
-    var invokeUrl = 'https://YOUR_API_GATEWAY_API_ID.execute-api.YOUR_PRIMARY_AWS_REGION.amazonaws.com/prod';
-    var endpoint = /(^https?:\/\/[^\/]+)/g.exec(invokeUrl)[1];
-    var pathComponent = invokeUrl.substring(endpoint.length);
+    let invokeUrl = `https://${window.config.restApiId}.execute-api.${window.config.region}.amazonaws.com/prod`,
```

Comment:
What's your browser support list? Do they all support `let` or are you transpiling?

Reply:
The behavior is whatever the default react-scripts build behavior was at the time this was built. :/

I'll eject it and check the settings.

Reply:
Eject confirms it's being babble'd.